src/assets/locales/fr.json

```diff
@@ -471,9 +539,11 @@
   "scene-url-modal.title": "URL d'une scène personnalisée",
   "scene-url-modal.url-input": "URL d'une scène",
   "select-input-field.placeholder": "Sélectionner...",
+  "share-popover.source.avatar-camera": "Caméra de l'avatar",
   "share-popover.source.camera": "Caméra",
   "share-popover.source.screen": "Écran",
   "share-popover.title": "Partager",
+  "sign-in-button": "Se connecter/Se déconnecter",
```

Comment:
Oups, really bad, it doesn't make sense. Fixed in https://github.com/mozilla/hubs/commit/5a43021cac0a58b37f433d2e0d35c5a5ff6cc5c3
"sign-in-button": "Se connecter/S'inscrire"

English is 
 "sign-in-button": "Sign in/Sign up"